x-pack/plugin/logsdb/src/yamlRestTest/resources/rest-api-spec/test/40_source_mode_setting.yml

```diff
@@ -609,239 +344,3 @@ modify final setting after index creation:
         body:
           index:
             mapping.source.mode: synthetic
-
----
-modify source mapping from stored to disabled after index creation:
```

Comment:
These tests are redundant, there's no way to configure source mode in mapping and updating the setting is covered in the test right above.